src/sentry/integrations/gitlab/integration.py

```diff
@@ -343,11 +353,16 @@ def build_integration(self, state):
         }
 
         user = get_user_info(data["access_token"], state["installation_data"])
-        group = self.get_group_info(data["access_token"], state["installation_data"])
-        include_subgroups = state["installation_data"]["include_subgroups"]
         scopes = sorted(GitlabIdentityProvider.oauth_scopes)
         base_url = state["installation_data"]["url"]
 
+        if state["installation_data"].get("group", None):
```

Comment:
@ceorourke I'm not a python guy, just went for the rebase. I googled this might be the way (?)


> Are you sure this won't result in a KeyError if there is no group?
> https://github.com/getsentry/sentry/pull/15188/files#r418253429

https://github.com/getsentry/sentry/pull/25824#issuecomment-1021547972

Reply:
Yep `.get()` is what you want here - though the default is already `None` so adding it is redundant.